File: src/gatling/scala/org/openapitools/client/model/OfferConditionDTO.scala

package org.openapitools.client.model


case class OfferConditionDTO (
    _type: Option[OfferConditionType],
    _quality: Option[OfferConditionQualityType],
    /* Описание товара. Подробно опишите дефекты, насколько они заметны и где их искать.  */
    _reason: Option[String]
)
object OfferConditionDTO {
    def toStringBody(var_type: Object, var_quality: Object, var_reason: Object) =
        s"""
        | {
        | "type":$var_type,"quality":$var_quality,"reason":$var_reason
        | }
        """.stripMargin
}

File: ympa_fsharp_giraffe_server/src/model/OfferConditionDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.OfferConditionQualityType
open ympa_fsharp_giraffe_server.Model.OfferConditionType

module OfferConditionDTO =

  //#region OfferConditionDTO


  type OfferConditionDTO = {
    Type : OfferConditionType;
    Quality : OfferConditionQualityType;
    Reason : string;
  }
  //#endregion
